remme/rpc_api/__main__.py

```diff
@@ -0,0 +1,76 @@
+# Copyright 2018 REMME
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ------------------------------------------------------------------------
+
+import argparse
+import asyncio
+import logging
+
+from aiohttp import web
+import aiohttp_cors
+
+from zmq.asyncio import ZMQEventLoop
+from remme.shared.logging import setup_logging
+
+from remme.shared.stream import Stream
+from remme.settings import cfg_rpc, ZMQ_URL
```

Comment:
Here we need to remove this from `remme.settings` module initialization because it fails to load when it cannot found the desired config. This behavior is pretty hard to debug when the module won't load.